openapiart/bundler.py

```diff
@@ -336,8 +345,15 @@ def _validate_names(self, regex, components_key, components):
                             "%s property name `%s` contains invalid characters"
                             % (key, name)
                         )
+                    upper_case = self._check_upper_case(name)
+                    if upper_case:
```

Comment:
would suggest directly using calling the method here if the value is not needed anywhere else

Reply:
done